MdePkg/Include/IndustryStandard/ServiceProcessorManagementInterfaceTable.h

```diff
@@ -95,4 +97,17 @@ typedef struct {
 
 #pragma pack()
 
+///
+/// SPMI Revision (defined in spec)
+///
+#define EFI_ACPI_SERVICE_PROCESSOR_MANAGEMENT_INTERFACE_TABLE_REVISION  0x05

```

Comment:
> I assume there should only be only one IPMI per platform, so linking the two objects with a Token should not be > necessary right ?

Reply:
> Maybe it would be good to have `5` in the macro name, like `EFI_ACPI_SERVICE_PROCESSOR_MANAGEMENT_INTERFACE_TABLE_REVISION_5` for instance. If a new revision comes out, the code will not be automatically compatible to the latest revision.

I think the spec mandate it to be '5', I dont think its incremental revision.
In my opinion, the ACPI SPMI generator needs to modify and set the maximum supported revision as '5'.

Reply:
Some ACPI table generator are able to support multiple revisions (e.g. Madt Generator).
If the revision is not mandated in the macro name, it becomes hard to specify the min/max revision the generator can support